src/main/webapp/view/discussion/discussion.js

```diff
@@ -147,6 +154,30 @@ function updateNewCommentTimestamp(timeMilliseconds) {
   newCommentTimestampMilliseconds = timeMilliseconds;
 }
 
+/**
+ * Returns an array of the {@code DiscussionComment}s within
+ * {@code TIME_TOLERANCE_MILLISECONDS} milliseconds to the {@code
+ * timestampMilliseconds}. This returns an empty array if no elements are
+ * nearby.
+ */
+function getNearbyDiscussionComments(timestampMilliseconds) {
+  const nearby = [];
+  // currentRootDiscussionComments is already sorted by timestamp.
+  for (const element of currentRootDiscussionComments) {
```

Comment:
When we have a lot of comments, I feel it is inefficient to loop through all of the current root comments like so. 

Do you think you can look into using binary search to find the closest element to the lower time bound and start iterating from there instead?

Reply:
I just tested with 1000 comments (our goal in design doc) at timestamp 00:00.  

It takes 1.5 milliseconds to run `getNearbyDiscussionComments(0)` and return an array with them all.  So this should be fine, especially as usually the comments wont all be grouped at 00:00 and only a few would be added to the array.

Reply:
Sounds good. Thanks for looking into that!